src/python/ensembl/utils/rloader.py

```diff
@@ -0,0 +1,70 @@
+# See the NOTICE file distributed with this work for additional information
+#   regarding copyright ownership.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""
+Allow to seamlessly load / read the content of a remote file as if it was located locally.
+
+@author: Marc Chakiachvili
+
+"""
```

Comment:
As far as I can remember, authorship is already given by the commit/PR, so we shouldn't leave it in the docstrings that will be published:
```suggestion
"""Allow to seamlessly load / read the content of a remote file as if it was located locally."""
```
An example on how to use the contents of this would be useful as well.